libs/wire-subsystems/test/unit/Wire/MockInterpreters/UserSubsystem.hs

```diff
@@ -1,15 +1,11 @@
 module Wire.MockInterpreters.UserSubsystem where
 
-import Data.Qualified
 import Imports
 import Polysemy
 import Wire.API.User
-import Wire.UserKeyStore
 import Wire.UserSubsystem
 
 userSubsystemTestInterpreter :: [UserAccount] -> InterpreterFor UserSubsystem r
-userSubsystemTestInterpreter initialUsers =
+userSubsystemTestInterpreter _initialUsers =
   interpret \case
-    GetLocalUserAccountByUserKey localUserKey -> case (tUnqualified localUserKey) of
-      EmailKey _ email -> pure $ find (\u -> userEmail u.accountUser == Just email) initialUsers
     _ -> error $ "userSubsystemTestInterpreter: implement on demand"
```

Comment:
why was this removed?

Reply:
as you requested in a TODO, this is not an operation of the user subsystem effect any more.  if it was used in any tests, they will fail and we'll look into that.